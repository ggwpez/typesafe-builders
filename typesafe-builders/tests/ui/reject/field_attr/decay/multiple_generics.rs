mod other {
	use typesafe_builders::prelude::*;

	#[derive(Builder)]
	pub struct Struct {
		#[builder(decay)]
		x: Result<(), ()>,
	}
}

fn main() {
}
