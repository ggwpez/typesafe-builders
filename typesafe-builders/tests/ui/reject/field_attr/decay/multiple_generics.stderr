error: Need exactly one generic argument, but got 2
 --> tests/ui/reject/field_attr/decay/multiple_generics.rs:7:13
  |
7 |         x: Result<(), ()>,
  |                   ^^^^^^
